test/test_muscles.cpp

```diff
@@ -1200,6 +1200,15 @@ TEST(Muscles, deepCopy)
     EXPECT_STREQ(shallowCopy.muscleGroup(0).name().c_str(), "muscleGroupName");
     EXPECT_STREQ(deepCopyNow.muscleGroup(0).name().c_str(), "muscleGroupName");
     EXPECT_STREQ(deepCopyLater.muscleGroup(0).name().c_str(), "muscleGroupName");
+
+    // Set new name
+    muscles.muscleGroup(0).setName("newMuscleGroupName");
+
+    //Check if copies changed names
+    EXPECT_STREQ(muscles.muscleGroup(0).name().c_str(), "newMuscleGroupName");
+    EXPECT_STREQ(shallowCopy.muscleGroup(0).name().c_str(), "newMuscleGroupName");
+    EXPECT_STREQ(deepCopyNow.muscleGroup(0).name().c_str(), "newMuscleGroupName"); 
+    EXPECT_STREQ(deepCopyLater.muscleGroup(0).name().c_str(), "newMuscleGroupName"); 
```

Comment:
That should not happen. Deepcopy should keep its previous name

Reply:
Make sure the "newMuscleGroupName" is the previous name of the muscle. If the test pass like this, it means there is a bug! I'll correct it :)

Reply:
"muscleGroupName" is the previous name of the muscle. I changed it to "newMuscleGroupName" for it to pass, although it's not suppose to pass.